module_utils/kafka_lib_commons.py

```diff
@@ -80,6 +80,10 @@
 
     replica_factor=dict(type='int', required=False, default=0),
 
+    force_reassign=dict(type='bool', required=False, default=False),
```

Comment:
can those 2 new params be also be added to the `DOCUMENTATION_COMMON` variable?

Reply:
Done :)